manifests/cluster/patch/pg-configmap.yaml

```diff
@@ -0,0 +1,44 @@
+- op: add
```

Comment:
You just need to add the items not in the original list, no need to add duplicated entries.